effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -86,9 +86,9 @@ object Transformer {
       case machine.Jump(label) =>
         emit(Comment("statement jump"))
         shareValues(label.environment, Set())
-        storeEnvironment(initialEnvironmentPointer, label.environment)
 
-        emit(TailCall(transform(label), List(LocalReference(environmentType, "environment"), getStackPointer())))
+        val arguments = label.environment.map(transform)
+        emit(Call("", Tailcc(), VoidType(), transform(label), arguments :+ getStackPointer()))
```

Comment:
Shouldn't it be at least `"_"`? It feels wrong to have an empty string here, even though void does not return.

Reply:
Done